cloud/aws/templates/aws_oidc/main.tf

```diff
@@ -22,7 +28,8 @@ resource "aws_db_instance" "civiform" {
     Name = "${var.app_prefix} Civiform Database"
     Type = "Civiform Database"
   }
-
+  availability_zone               = data.aws_availability_zones.available.names
+  deletion_protection             = local.deletion_protection
```

Comment:
I'm guessing it means that it will be impossible to run `destroy` on prod environments, right? Should we fail `destroy` early if civiform_mode is prod? 
I think we can suggest either
1. Print something like "destroying of prod environment is not supported. Get a cup of coffee and prepare for 1h long manual deleting of stuff in aws console."
2. Go unmark database (and probably few other places) as deletion-protected and then run destroy script again.

I don't think we need to do anything at this point. Just thinking.

Reply:
Correct. destroy is meant to be there for testing only. Prod should be HARD to destroy. I like your "get a cup of coffee" idea, I'll add it to destroy script.